contents/docs/getting-started/group-analytics.mdx

```diff
@@ -1,70 +1,9 @@
 ---
-title: Group analytics
```

Comment:
I feel strongly that we shouldn't rename this. Groups is a very generic term, group analytics explains it a lot better. We mention group analytics everywhere already, and importantly, on the pricing page. It's also better for SEO. 

Groups could also refer to user groups or any other type of group.

Reply:
Also for the Algolia search, groups vs group analytics, I'm pretty sure we can set up an override that lets us show group analytics for the term "groups". Do you know @smallbrownbike ?

Reply:
I think if we can override the algolia search, then thats a good compromise. My main concern is that we refer to it as "Groups" in the app, so people are more likely to search for "groups" and not "group analytics"

@corywatilo maybe you know how to override?